buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+val jdk8Modules = setOf(
```

Comment:
same exists in `KtorBuildProperties` - is it really needed to duplicate it?

Reply:
Thanks, removed